features/path_rules.feature

```diff
@@ -201,9 +201,3 @@ Feature: Path rules
     Then the response status-code must be 200
     And the response must be served by the "aaa-slash-bbb-slash-prefix" service
     And the request path must be "/aaa/bbb/"
-
-  Scenario: An Ingress with a trailing slashes in an exact path rule should not match requests without a trailing slash
```

Comment:
Trailing slashes is like regular expressions, proxies behave differently and expect or not a trailing slash.

Reply:
The published docs and the KEP actually define this behavior. They don't let it open to interpretation by the ingress-controller.

5th row in https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/20190125-ingress-api-group.md#examples